examples/classification/main.py

```diff
@@ -250,6 +264,7 @@ def train(config, compression_ctrl, model, criterion, is_inception, lr_scheduler
 
             for key, value in stats.items():
                 if isinstance(value, (int, float)):
+                    mlflow.log_metric("compression/statistics/{0}".format(key), value, step=len(train_loader) * epoch)
```

Comment:
Step numbers in iterations are going to be very large for datasets like ImageNet. Maybe add a flag to write steps as epoch counts.

Reply:
Done